libraries/RW/K8s/k8sutils.py

```diff
@@ -0,0 +1,78 @@
+"""
+K8s util library for k8s specific formatting and other tools.
+
+Scope: Global
+"""
+import re, kubernetes, yaml, logging, json, jmespath
+from struct import unpack
+import dateutil.parser
+from benedict import benedict
+from typing import Optional, Union
+from RW import platform
+from RW.Utils import utils
+from enum import Enum
+
+logger = logging.getLogger(__name__)
+
+class K8sUtils:
+    """
+    K8s utils to help format K8s output.
+    """
+
+    ROBOT_LIBRARY_SCOPE = "GLOBAL"
+    
+    def convert_to_metric(self, 
+        command: str=None,
```

Comment:
Thinking on this further, attempting to use K8s.Shell in this utils class would cause some unruly dependencies/side effects. I'll take a look at the connection mixin shortly to see if I can make it safer/easier to use like this.

Reply:
Not sure I follow the second comment... let's chat live. I was struggling a bit with how to best structure things like utilities that aren't just something like namespace specific, but ended up here to make both k8s-kubectl-get and this codebundle work.